app/src/main/res/layout/component_list_content.xml

```diff
@@ -3,17 +3,27 @@
   xmlns:app="http://schemas.android.com/apk/res-auto"
   android:layout_width="match_parent"
   android:layout_height="wrap_content"
-  android:orientation="vertical">
+  xmlns:tools="http://schemas.android.com/tools"
+  android:paddingStart="@dimen/bpkSpacingLg"
+  android:paddingEnd="@dimen/bpkSpacingLg"
+  android:background="?attr/selectableItemBackground"
+  android:orientation="horizontal">
```

Comment:
this is why i ended up making the divider change here too - this is just a linearlayout right now